eth/contracts/Most.sol

```diff
@@ -100,6 +100,12 @@ contract Most is
     //
     // Tx emits a CrosschainTransferRequest event that the relayers listen to
     // & forward to the destination chain.
+
+    // let's use modern standards.
```

Comment:
can you clarify this comment a bit, I'm not sure I follow.

Regardless of whether the allowance is increased via `approve` or `permit` the end result is the same - Most can spend up to a `value` on behalf of the sender, so it's more of a concern for the fronted to distinguish if token implement the ERC and whether to ask the user to sign a `permit` or just a plain `approval` tx ?

Reply:
`permit` allows someone to execute `approve` on behalf of someone else. For the bridge frontend client, it might not be that useful. But for some users of bridge, who want to cross-contract call the bridge it might be very useful.